ONSdigital/rm-collection-exercise-service
# Update action rules on update event

# Motivation and Context
Currently the action rules for the matching events do not get updated
when event dates get updated.

# What has changed



This PR adds the logic to update action rules for all business survey events.

# How to test?



Use UI to update mps or go_live events datetime and see the matching action rule trigger datetime get updated as well in the actionrule table.

# Links



https://trello.com/c/r79j0Xlg/125-tbl123-link-action-rules-to-action-plans-at-create-time-update
